proto/buf/alpha/registry/v1alpha1/admin.proto

```diff
@@ -35,6 +35,35 @@ message BreakingChangePolicy {
   BreakingChangeCategory category = 3;
 }
 
+message UniquenessPolicy {
+  optional bool enabled = 1;
```

Comment:
assuming `enabled` entails turning on uniqueness for both FQN && filepaths, right?

Reply:
yeah, as far as I understand it, there's no ask to have these enabled separately.
I think the idea is that typical proto uniqueness enforcement, like FQN collisions and filename collisions are all avoided at once